05 Presentational Comp/src/router.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
-import { Router, Route, IndexRoute, hashHistory } from 'react-router';
+import { Route, BrowserRouter } from 'react-router-dom';
 import { App } from './app';
 import { About, MembersPage } from './components';
 
 export const AppRouter: React.StatelessComponent<{}> = () => {
   return (
-    <Router history={hashHistory}>
-      <Route path="/" component={App} >
-        <IndexRoute component={About} />
+    <BrowserRouter>
```

Comment:
Since the old version used hashHistory, should it not be HashRouter instead of BrowserRouter?

Reply:
Absolutely right. I'll change it